Error: Cannot find module '@oclif/errors'

<!--
  Thanks for filing an issue on Apollo Tooling!

  Please make sure that you include the following information to ensure that your issue is actionable.

  If you don't follow the template, your issue may end up being closed without anyone looking at it carefully, because it is not actionable for us without the information in this template.
-->

When using a strict package manager such as pnpm (or yarn pnp), running `apollo` will fail with error `Error: Cannot find module '@oclif/errors'`. This is because node cannot [resolve the `@oclif/errors` import](https://github.com/apollographql/apollo-tooling/blob/80a715dc2396b79278c8006e024b5393af83175d/packages/apollo/bin/run#L4) due to pnpm's strict `node_modules` structure, since `@oclif/errors` is not declared in package.json "dependencies".

**Intended outcome:**

No fatal error when launching the Apollo CLI.

<!--
What you were trying to accomplish when the bug occurred, and as much code as possible related to the source of the problem.
-->

**Actual outcome:**

```
> apollo client:codegen --config=./apollo.config.js --target=typescript --outputFlat ./graphql-types.ts --watch

internal/modules/cjs/loader.js:583
    throw err;
    ^

Error: Cannot find module '@oclif/errors'
    at Function.Module._resolveFilename (internal/modules/cjs/loader.js:581:15)
    at Function.Module._load (internal/modules/cjs/loader.js:507:25)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/Users/spencerelliott/Dev/Yolk-HQ/yolkjs/common/temp/node_modules/.registry.npmjs.org/@oclif/config/1.12.0/node_modules/@oclif/config/lib/config.js:3:18)
    at Module._compile (internal/modules/cjs/loader.js:689:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:700:10)
    at Module.load (internal/modules/cjs/loader.js:599:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:538:12)
    at Function.Module._load (internal/modules/cjs/loader.js:530:3)
```

<!--
A description of what actually happened, including a screenshot or copy-paste of any related error messages, logs, or other output that might be related. Places to look for information include your browser console, server console, and network logs. Please avoid non-specific phrases like “didn’t work” or “broke”.
-->

**How to reproduce the issue:**

```shell
npm install --global pnpm
pnpm install apollo
./node_modules/.bin/apollo
```

<!--
If possible, please create a reproduction and link to it here. If reporting a CLI issue, it's helpful to open a sample repository where your broken commands can be run against to reproduce the error. You can fork the [fullstack tutorial](https://github.com/apollographql/fullstack-tutorial) to reproduce many issues.

Provide instructions for how the issue can be reproduced by a maintainer or contributor. Be as specific as possible, and only mention what is necessary to reproduce the bug. If possible, try to isolate the exact circumstances in which the bug occurs and avoid speculation over what the cause might be.
-->

**Versions**

```shell
$ npm -v
6.8.0
$ pnpm -v
2.25.6
$ node -v
v10.15.0
```

<!--
Please add the version of the extension/cli/tool that you are experiencing issues with. Without a version, it may not be possible to reproduce your issue.
-->

--- a/CHANGELOG.md
+++ b/CHANGELOG.md
@@ -5,6 +5,7 @@
 - `apollo`
   - Use generated Typescript types via client:codegen [#1016](https://github.com/apollographql/apollo-tooling/pull/1016)
   - Remove default `--tag=current` for some client commands that used it [#1062](https://github.com/apollographql/apollo-tooling/pull/1062)
+  - Add missing dependency `@oclif/errors` [#1068](https://github.com/apollographql/apollo-tooling/pull/1068)
 
 # `apollo@2.5.3`
 

--- a/packages/apollo/package.json
+++ b/packages/apollo/package.json
@@ -34,6 +34,7 @@
     "@apollographql/apollo-tools": "file:../apollo-tools",
     "@oclif/command": "1.5.10",
     "@oclif/config": "1.12.0",
+    "@oclif/errors": "1.2.2",
     "@oclif/plugin-autocomplete": "0.1.0",
     "@oclif/plugin-help": "1.2.11",
     "@oclif/plugin-not-found": "1.2.2",

